Languages/german.lang

```diff
@@ -285,7 +285,7 @@ msgid "Block IOS Reload"
 msgstr "Blocke IOS-Neuladen"
 
 msgid "Block Loader Layout Button"
-msgstr ""
+msgstr "Blocke Loader Layout Button"
```

Comment:
`Loader Layout Button` is not Deutsch.

Reply:
I know, I could not find a better word for it...

Reply:
Well, @wiidev might explain what `Block Loader Layout Button"` actualy does /mean.

Reply:
The translations below pause use a similar wording. That’s why I translated it that way.